nshlib/nsh_printf.c

```diff
@@ -84,7 +87,34 @@ int cmd_printf(FAR struct nsh_vtbl_s *vtbl, int argc, char **argv)
             {
               case 'x':
```

Comment:
should we use something like this for multiple char:
printf \x12 \x34 \x56 \x78
from https://wiki.bash-hackers.org/commands/builtin/printf:
one /x should generate one byte output.

Reply:
Hi @xiaoxiang781216, using "printf \x01 \x00 \x00 \x00 > /dev/userleds" works, but the user will need to take care about endianess. Using \x00000001 is more intuitive, because we know the LSB is 1